orderbook/src/fee.rs

```diff
@@ -168,7 +168,7 @@ impl MinFeeCalculator {
         amount: Option<U256>,
         kind: Option<OrderKind>,
     ) -> Result<U256, PriceEstimationError> {
-        let gas_price = self.gas_estimator.estimate().await?;
+        let gas_price = self.gas_estimator.estimate().await?.estimate();
```

Comment:
Seems weird to need to `.estimate()` twice... I can only assume these are two different functions, but which is which.

Reply:
Good point. I think the second estimate is to turn the estimated gas price into a legacy `gas_price` value. Maybe something like `to_legacy_gas_price()` might be more descriptive?

Reply:
Changed to `effective_gas_price()`, since this term is broadly used and known in ethereum protocol and defi projects.

Reply:
`effective_gas_price` sounds good to me.